runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -243,6 +252,37 @@ void update(boolean updated, boolean paused, XpSnapshotSingle xpSnapshotSingle)
 		SwingUtilities.invokeLater(() -> rebuildAsync(updated, paused, xpSnapshotSingle));
 	}
 
+	void toggleHeaderPanel(JPanel progressWrapper)
+	{
+		int progressSkillIconPosition = 0;
+		boolean progressSkillIconState = progressWrapper.getComponent(progressSkillIconPosition).isVisible();
+
+		if (headerPanel.isVisible())
+		{
+			progressWrapper.setBorder(new EmptyBorder(5, 1, 5, 5));
+		}
+		else
+		{
+			progressWrapper.setBorder(DEFAULT_PROGRESS_WRAPPER_BORDER);
+		}
+
+		headerPanel.setVisible(!headerPanel.isVisible());
+		progressWrapper.getComponent(progressSkillIconPosition).setVisible(!progressSkillIconState);
+	}
+
+	private JLabel getSkillIcon(SkillIconManager iconManager, Skill skill, int width, int height, boolean small)
+	{
+		JLabel skillIcon = new JLabel();
+		Image image = small ? iconManager.getSkillImage(skill, true) : iconManager.getSkillImage(skill);
+
+		skillIcon.setIcon(new ImageIcon(image));
```

Comment:
```suggestion

		skillIcon.setIcon(new ImageIcon(iconManager.getSkillImage(skill, small)));
```

Reply:
I did modify [Line 208](https://github.com/runelite/runelite/pull/13065/files/a2866ce87fa911950d78083988cb0f6ba5ec102a#diff-3f7bdad15fc0f983fae5f9172f851786bc503296baf07e81bb7e10a392fbecb5R208) to use this function, but get the full sized image instead of the `small`.

Reply:
I see that, my point is that `SkillIconManager#getSkillImage(Skill)` is an overload which resolves to `SkillIconManager#getSkillImage(Skill, false)`, so `small` can be passed as the second argument to the latter signature instead of using a ternary to indirectly pass `false` to it.

Reply:
Yeah you're definitely right... duh.